techniques/system/common/1.0/promises.st

```diff
@@ -47,11 +70,17 @@ bundle common va
 {
   vars:
 
-      "end" slist => { "restart_services", "endExecution" };
+      "end" slist => {
+&if(!INITIAL)&
+        "restart_services", 
+&endif&
```

Comment:
OK, this is used to restart syslog services, so initial promises don't need it. Got it.